client/src/entrypoints/admin/core.js

```diff
@@ -32,6 +33,29 @@ window.wagtail = wagtail;
 
 window.escapeHtml = escapeHtml;
 
+/**
+ * Support legacy global URLify which can be called with `allowUnicode` as a third param.
+ * Was not documented and only used in modeladmin prepopulate JS.
+ *
+ * @deprecated RemovedInWagtail70
+ * @see https://github.com/django/django/blob/main/django/contrib/admin/static/admin/js/urlify.js#L156
+ *
+ * @param {string} str
+ * @param {number} numChars
+ * @param {boolean} allowUnicode
+ * @returns {string}
+ */
+window.URLify = (str, numChars, allowUnicode) =>
+  allowUnicode
+    ? str
+        .toLowerCase()
+        .replace(/\[^-_\p{L}\p{N}\s]/g, '')
+        .replace(/^\s+|\s+$/g, '')
+        .replace(/[-\s]+/g, '-')
+        .substring(0, numChars)
+        .replace(/-+$/g, '')
+    : urlify(str, numChars);
```

Comment:
We can reuse the implementation as-is, can't we?

```suggestion
window.URLify = URLify;
```

Reply:
yep thanks, I missed this.